resources/tentative/todomvc/src/components/todo-item/todo-item.template.js

```diff
@@ -0,0 +1,23 @@
+const template = document.createElement("template");
+const globals = "src/styles/global.css";
+const styles = "src/styles/todo-item.css";
+
+template.id = "todo-item-template";
+template.innerHTML = `
+    <link rel="stylesheet" href="${globals}" />
+    <link rel="stylesheet" href="${styles}" />
+    <li class="todo-item">
+        <div class="display-todo">
+            <label class="toggle-todo-label visually-hidden">Toggle Todo</label>
```

Comment:
Shouldn't this specify "for"?

Reply:
once the "id" attribute of the component is assigned, node ids and "for" attributes are set. 
https://github.com/flashdesignory/Speedometer/blob/todo-wc/resources/tentative/todomvc/src/components/todo-item/todo-item.component.js#L46

Reply:
Example of rendered output:
<img width="982" alt="Screenshot 2023-06-23 at 12 55 45 PM" src="https://github.com/WebKit/Speedometer/assets/372973/42044900-f645-42ce-bc66-83f7d9bd3d55">

Reply:
That seems rather unusual way for shadow DOM to operate. IDs in each shadow tree lives in a different namespace so we should be able to just say `id="toggle-todo"` and `for="toggle-todo"` and be done with it. Same goes for edit-todo and remove-todo as well. These can all be "edit-todo" and "remove-todo".

Reply:
done!